modules/programs/jq.nix

```diff
@@ -0,0 +1,68 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.programs.jq;
+
+  colorType = mkOption {
+    type = types.str;
+    description = "ANSI color definition";
+    example = "1;31";
+  };
+
+  colorsType = types.submodule {
+    options = {
+      null    = colorType;
+      false   = colorType;
+      true    = colorType;
+      numbers = colorType;
+      strings = colorType;
+      arrays  = colorType;
+      objects = colorType;
+    };
+  };
+
+in
+
+{
+  options = {
+    programs.jq = {
+      enable = mkEnableOption "jq command-line JSON processor";
+      colors = mkOption {
```

Comment:
Generally I prefer to have blank lines between options to help spot them easier.